plugins/outputs/influxdb_v2/README.md

```diff
@@ -30,6 +30,15 @@ The InfluxDB output plugin writes metrics to the [InfluxDB v2.x] HTTP service.
   ## If true, the bucket tag will not be added to the metric.
   # exclude_bucket_tag = false
 
+  ## If true, no create bucket requests will be sent. Set to true when using
+  ## Telegraf with a user without permissions to create buckets or when the
+  ## the bucket already exists
+  # skip_bucket_creation = false
+
+  ## Default retention rule for any newly created buckets. If unset, the default
+  ## retention rule is forever. Ignored if skip_bucket_creation is false.
+  # default_bucket_retention = "0s"
```

Comment:
The v1 plugin also lets you specify the retention policy in a metric tag, not just a toml setting (see `retention_policy_tag` and `exclude_retention_policy_tag`). I wonder if we need them here. I don't know how often they're used and how necessary they are.

Do you think we need to add similar functionality to get the retention policy from the metric?

Reply:
The retention policy is only needed when the bucket is created. The tag would only be necessary if the user was using `bucket_tag` and wanted different retention policies for buckets that did not already exist.

It would be trivial to implement. However, I'm not sure it's worth having a tag on every metric for the retention policy when it may never be used. Though I guess that decision could be left up to the user.

Reply:
I think the users who use v1's retention_policy_tag do have them on every metric.

Let's not add the retention policy tags functionality in this PR. If there are users who need it with v2, they will file an issue and we can add it in the future.